documentation update for cms specific options to category (#917)
capitalone/DataProfiler: docs/source/profiler.rst

      * top_k_categories - (int) Number of categories to be displayed when reporting
      * max_sample_size_to_check_stop_condition - (int) The maximum sample size before categorical stop conditions are checked
      * stop_condition_unique_value_ratio - (float) The highest ratio of unique values to dataset size that is to be considered a categorical type

>       * cms - (Boolean) Enables or Disables the use of count min sketch / heavy hitters for approximate frequency counts.

Reviewer: oh interesting... I was expecting `CMS` to be its own options class .... 🤔

Author: yup it's a method for doing approximate frequency counts. Only intended for the categorical profile for now.

>       * cms_confidence (float) Defines the number of hashes used in CMS.

Reviewer: ```suggestion
      * cms_confidence - (float) Defines the number of hashes used in CMS.
```

>         eg. confidence = 1 - failure probability, default 0.95

Reviewer: does this need a tab for rendering?

Author: I don' tthink so?

>       * cms_confidence - (float) Defines the number of hashes used in CMS.
>         eg. confidence = 1 - failure probability, default 0.95

Reviewer: ```suggestion
      * cms_confidence - (float) Defines the number of hashes used in CMS. E.G. confidence = 1 - failure probability, default 0.95
```

>       * cms_relative_error: (float) Defines the number of buckets used in CMS,

Reviewer: ```suggestion
      * cms_relative_error - (float) Defines the number of buckets used in CMS,
```

>       * cms_relative_error - (float) Defines the number of buckets used in CMS,
>             default 0.01

Reviewer: ```suggestion
      * cms_relative_error - (float) Defines the number of buckets used in CMS, default 0.01
```

>       * cms_max_num_heavy_hitters (int) The value used to define

Reviewer: ```suggestion
      * cms_max_num_heavy_hitters - (int) The value used to define
```

>         the threshold for minimum frequency required by a category to be counted

Reviewer: indent these for clarity where they fall?

Reviewer: maybe don't indent if its all attributes under `CategoricalOptions`

Author: following pattern of pre-existing options at the same level
